src/parsers/dict_to_struct.jl

```diff
@@ -188,8 +188,8 @@ function gen_dict_parser(dict::Dict{String,Any})
     for (gen_type_key,gen_type_dict) in dict
         if gen_type_key =="Thermal"
             for (thermal_key,thermal_dict) in gen_type_dict
-                push!(Generators,ThermalDispatch(convert(String,thermal_dict["name"]),
```

Comment:
Do we need a conversion here? The name should be a String. Many conversions might slow down the process.

Reply:
It is possible that these are not required. I'll run the tests again without some of these conversions, and if they pass I'll commit the changes and merge this.